packages/atomic/src/pages/examples/folding.html

```diff
@@ -76,6 +86,365 @@
+                      border-radius: 4px;
+                    }
+
+                    .slack-atomic-result-icon {
+                      flex-shrink: 0;
+                      height: 32px;
+                      width: 32px;
+                      background-size: cover;
+                      margin-right: 16px;
+                    }
+
+                    .slack-msg-result-section-title {
+                      display: flex;
+                      background-color: #f6f7f9;
+                      border: 1px solid #f6f7f9;
+                      border-radius: 8px;
+                      flex-grow: 1;
+                      padding: 8px 8px 0px 8px;
+                    }
+
+                    .slack-link {
+                      color: #1372ec;
+                      font-size: 12px;
+                    }
+
+                    .slack-result-date {
+                      color: #616061;
+                      box-sizing: border-box;
+                      font-style: normal;
+                      font-weight: 400;
+                      font-size: 12px;
+                    }
+
+                    .slack-result-thread-parent {
+                      display: flex;
+                      background-color: #f6f7f9;
+                      border-radius: 8px;
+                      flex-grow: 1;
+                      padding-bottom: 16px;
+                    }
+
+                    .slack-result-author-name {
+                      font-size: 16px;
+                      margin-right: 4px;
+                      font-weight: bold;
+                    }
+
+                    .slack-result-thread-bottom-text {
+                      font-style: normal;
+                      font-weight: 700;
+                      font-size: 12px;
+                      color: #626971;
+                    }
+
+                    .slack-result-section-excerpt {
+                      display: flex;
+                      margin-left: 50px;
+                    }
+
+                    .slack-result-fields {
+                      margin: 10px 0 0 50px;
+                    }
+
+                    .slack-result-normal-top {
+                      margin-top: 16px;
+                    }
+
+                    .slack-result-thread-top {
+                      margin-top: 8px;
+                    }
+
+                    .slack-result-msg-excerpt {
+                      margin: 4px 0 0 0;
+                    }
+
+                    .slack-others-result-section-title {
+                      font-size: 18px;
+                      color: black;
+                    }
+
+                    .slack-result-footer {
+                      display: flex;
+                      justify-content: space-between;
+                    }
+
+                    ::part(button) {
+                      margin: 8px 0 0 50px;
+                    }
+
+                    ::part(show-hide-button) {
+                      margin: 8px 0 0 50px;
+                    }
```

Comment:
I think it would be easier to understand what these parts refer to if we preceded them with their tag.